core-api/src/main/java/com/sequenceiq/cloudbreak/api/model/ldap/LdapConfigBase.java

```diff
@@ -206,6 +209,14 @@ public void setUserDnPattern(String userDnPattern) {
         this.userDnPattern = userDnPattern;
     }
 
+    public String getCertificate() {
+        return certificate;
+    }
+
+    public void setCertificate(String certificate) {
```

Comment:
The main problem is that line brakes are making the JSON invalid, either you ask for \n or send it in base64 like in case of blueprint or Gateway:

   @ApiModelProperty(GatewayModelDescription.KNOX_SSO_CERT)
    @JsonSerialize(using = Base64Serializer.class)
    @JsonDeserialize(using = Base64Deserializer.class)
    private String tokenCert;

Reply:
It is sent in and stored in base64 encoded form (see in CLI), and decoded by Salt only when needed with file.decode